librz/bin/format/java/class_bin.c

```diff
@@ -42,7 +42,8 @@ static char *java_class_constant_pool_stringify_at(RzBinJavaClass *bin, ut32 ind
 }
 
 static ut32 sanitize_size(st64 buffer_size, ut32 count, ut32 min_struct_size) {
-	ut64 memory_size = count * min_struct_size;
+	ut64 memory_size = count;
+	memory_size *= min_struct_size;
```

Comment:
why this?

Reply:
Ok got it on mattermost chat.

I suggest `ut64 memory_size = (ut64)count * min_struct_size` to make it clearer that the cast to ut64 is important. Otherwise it's easily overseen.